### ISSUE TITLE
ZLYA VAC Review (DR1)

### ISSUE THREAD

**Issue by dylanagreen** [desihub/desida#11]
Draft directory:`/global/cfs/cdirs/desi/users/abrodze/QSOredshiftsY1/zlya-QSO-DR1-v0`
1 catalog file, 3 files total

Initial Checks:
- [x] Includes README
- [x] Columns in ALLCAPS
- [x] Extension names in ALLCAPS
- [x] Files include units

Initial Notes:
- Can you draft a short (~1-2 paragraph) summary of the VAC to include in the DR1 paper? For an example of what this would look like you can look at section 3.3.6 of the EDR paper (https://arxiv.org/abs/2306.06308).
- It looks like you have an .ipynb_checkpoints folder in your VAC directory, unless we need it I think we should delete it
- If possible can you propagate the degree units to the catalog (as outlined in the README)?
- No comments on the README at this time except to include paper links when they're available, although in the near future it'll need to be reformatted to match desidatadocs.

**Comment by abrodze** [comment]
Thanks for the review. The checkpoint file has been deleted and units are added. Text for the DR1 paper will be emailed.

**Comment by abrodze** [comment]
updated version directory: `/global/cfs/cdirs/desi/users/abrodze/quasarsY1/Y1-catalogs/VACs/zlya-QSO-DR1-v1`

**Comment by dylanagreen** [comment]
Thanks @abrodze, I've looked over the v2 files and I don't see any issues with them, and I see you've added the units. I think this VAC is almost ready to be signed off on and handed off for final checks, but there is one more thing I think needs to be taken care of.

I see that you have a jupyter notebook included in the VAC directory, do you want to release this as part of the VAC? If you do, we can certainly do that, but you'll need to do some minor (hopefully quick) things:

1. Make the notebook runnable by the general public. This means that the spectrumdir needs to be updated to refer to the public dr1 release, and not the internal desi-only access points, and that the reference to the RR templates also needs to be made public-acccesible (probably with some reference to a generic template_dir variable, with an included comment that the rr templates can be downloaded from https://github.com/desihub/redrock-templates)
2. Make a static html version of the notebook that can be viewed in a web browser (e.g. the AGN physical properties VAC from EDR has one hosted as part of their VAC here: https://data.desi.lbl.gov/public/edr/vac/edr/cigale/fuji/v1.4/CigaleExampleFuji.html)

**Comment by abrodze** [comment]
1. Notebook paths updated (please confirm this is what you meant)
2. HTML file added
3. README updated to match desidatadocs format
4. directory and catalog file renamed to (1) match the lowercase format I see for edr and (2) make it explicitly clear to an external user what the catalog is

**Comment by dylanagreen** [comment]
@abrodze I've looked over the README and jupyter notebooks and think they look good. The catalog file also looks good, except for one last question before we move this into final stages of review. The columns ERR_BI_CIV and ERR_AI_CIV have some infinity values. Nominally we prefer not to have any infinities or nans in the catalog however we can allow them if they're sufficiently documented in the README. Can you either address the infinities or document how one should interpret a value of infinity in the README for the associated columns? For example something like "a value of infinity means that CIV isn't found" (not accurate, just for illustration purposes).

**Comment by abrodze** [comment]
I added documentation on infinities to the readme

**Comment by dylanagreen** [comment]
@weaverba137 Let's move this to final review, with stubname `zlya` and version `v1.0`

**Comment by weaverba137** [comment]
I've copied the data to the VAC staging area. The original README file is temporarily retained as `README.md.bak`.

I just noticed that the `SURVEY` and `PROGRAM` fields are strings of length 4. This is fine if `main`/`dark` are the only possible values, but can you please verify that?

**Comment by dylanagreen** [comment]
@weaverba137 Yes, this VAC only contains objects from main/dark (per the README).

**Comment by weaverba137** [comment]
OK. I think that should be added to the description of these columns if it isn't already.

**Comment by dylanagreen** [comment]
If there's no objections now that the README is merged, this is ready to go to final location so we can close this off. Thanks

**Comment by weaverba137** [comment]
OK, done. Merge when ready.
